lldb/source/Expression/IRExecutionUnit.cpp

```diff
@@ -274,8 +274,9 @@ void IRExecutionUnit::GetRunnableInfo(Status &error, lldb::addr_t &func_addr,
 
   builder.setEngineKind(llvm::EngineKind::JIT)
       .setErrorStr(&error_string)
-      .setRelocationModel(triple.isOSBinFormatMachO() ? llvm::Reloc::PIC_
-                                                      : llvm::Reloc::Static)
+      .setRelocationModel((triple.isOSBinFormatMachO() || triple.isRISCV64())
```

Comment:
Should we add a comment here for RISCV?

Reply:
This is not need after large code model has been merged.